source/sqlsrv/conn.cpp

```diff
@@ -18,9 +18,12 @@
 //---------------------------------------------------------------------------------------------------------------------------------
 
 #include "php_sqlsrv.h"
+
+#ifndef __linux__
```

Comment:
Have you tried compiling without these headers on Windows? I've found they're not needed in the PDO_SQLSRV source.